### ISSUE TITLE
Typos LSP

### ISSUE THREAD

**Issue by oleflb** [HULKs/hulk#826]
@schluis uses (typos)[https://github.com/crate-ci/typos] in his nvim config which is a low-positive type checker.
Currently it does not have any false-positives in the hulk codebase, all actual findings have been resolved already.
Maybe we can integrate this tool into our CI. This should be straight-forward as it is also written in Rust.
It also allows to set a list of exceptions, in case we need this in the future.

**Comment by knoellle** [comment]
> ... all actual findings have been resolved already.

This is not true, for example `crates/hulk_webots/src/intertial_measurement_unit_devices.rs` has a superfluous `t` in the module name.
